### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by ale7714**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Change log
+
+## [Unreleased]
+
+### New features
+
+* Add `Gocov` support
```
Comment (GordonDiggs) [inline CHANGELOG.md]: Should we use  the `NEW` vs `FIX` format for these?

Comment (ale7714) [inline CHANGELOG.md]: I like that, and remove the `### New features`/`### Bug fixes` header since they're won't be needed anymore.

**Comment by ale7714** [comment]
@GordonDiggs updated :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Change log
+
+## [Unreleased]
```
Comment (GordonDiggs) [inline CHANGELOG.md]: I would ditch the brackets here

**Pull request merged**